examples/example_plugin_with_view_override/example_plugin_with_view_override/ui/FullWidthPage.jsx

```diff
@@ -1,7 +1,8 @@
+// TODO: This doesn't seem to be working presently
```

Comment:
Probably because we don't have the view-override plugin enabled by default - doh.

Reply:
Works fine after enabling the view-override plugin. Will correct.